akka-cluster/src/main/scala/akka/cluster/MembershipState.scala

```diff
@@ -119,4 +124,137 @@ import akka.annotation.InternalApi
       .map(_.uniqueAddress)
   }
 
+  def validNodeForGossip(node: UniqueAddress): Boolean =
+    node != selfUniqueAddress && isReachableExcludingDownedObservers(node)
```

Comment:
`isReachableExcludingDownedObservers` will not cover cross DC nodes, for them I think we should use `isReachable(selfUniqueAddress, node)`